### PR TITLE
Add Django 4.0 official documentation

### PR DESCRIPTION
**Pull request by suravshrestha**
## What does this PR do?
Add resources

### Description
Quick reference guide to Django provided by [codemy.com](https://codemy.com/) in PDF version
and official documentation on Django 4.0 in PDF version

### Why is this valuable (or not)?
The cheat sheet provides to quick start to create and run a Django project

### How do we know it's really free?
The cheat sheet provided by the author at his website codemy.com at https://codemy.com/books/django-quick-reference-guide.pdf
The documentation is provided by the official team.

### For book lists, is it a book? For course lists, is it a course? etc.
The first commit is includes a cheat sheet
The second commit link includes a documentation

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -327,6 +327,7 @@
 
 * [All in one Cheatsheet of Django](https://cheatography.com/ogr/cheat-sheets/django/pdf/) - Olivier R. (PDF)
 * [Django Quick Reference](https://www.codewithharry.com/blogpost/django-cheatsheet/) - Code with Harry (HTML)
+* [Django Quick Reference Guide](https://codemy.com/books/django-quick-reference-guide.pdf) - Codemy.com (PDF)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:330]: This looks more like a brochure for the paid courses, than a cheat sheet. Would it be a value add?

Comment (suravshrestha) [reply more/free-programming-cheatsheets.md:330]: The first pages of the book contain valuable quick reference to django.

Comment (eshellman) [reply more/free-programming-cheatsheets.md:330]: delete it

Comment (suravshrestha) [reply more/free-programming-cheatsheets.md]: @eshellman I've deleted the quick reference link

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**